datetime/src/main/java/com/afollestad/materialdialogs/datetime/DateTimePickerExt.kt

```diff
@@ -39,7 +39,7 @@ import com.afollestad.materialdialogs.utils.MDUtil.isLandscape
 import com.afollestad.materialdialogs.utils.MDUtil.resolveColor
 import java.util.Calendar
 
-typealias DateTimeCallback = ((dialog: MaterialDialog, datetime: Calendar) -> Unit)?
+typealias DateTimeCallback = ((dialog: MaterialDialog, datetime: Calendar?) -> Unit)?
```

Comment:
This parameter shouldn't be made nullable -- it'd be better to just not invoke the callback if the selected date is null with `?.let { }`

Reply:
Absolutely agree, the most important point is to avoid force cast `dateCallback?.invoke(it, getDatePicker().getDate()!!)`, for reduce possible problems